package.json

```diff
@@ -4,7 +4,7 @@
   "id": "lockbox@mozilla.com",
   "version": "0.1.4-alpha",
   "main": "dist/bootstrap.js",
-  "description": "A Lockbox extension for Firefox",
+  "description": "the simple way to store, retrieve and manage website login info",
```

Comment:
This really feels like it should have sentence case, even if the in-page string doesn't...